sdk/storage/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.12)
+
+project (azure-storage LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+find_package(LibXml2 REQUIRED)
+
+option(BUILD_STORAGE_SAMPLES "Build sample codes" ON)
+
+set(AZURE_STORAGE_HEADER
+    inc/blobs/blob.hpp
+    inc/blobs/internal/protocol/blob_rest_client.hpp
+    inc/common/internal/protocol/rest_client_utility.hpp
```

Comment:
This file has been removed.

Reply:
I am removing all the files/rebasing on a commit that has some valid files included. But the former approach would result in the CMake to report failure that no source file is specified for azure-storage.